airflow/providers/apache/livy/hooks/livy.py

```diff
@@ -173,7 +173,7 @@ def post_batch(self, *args: Any, **kwargs: Any) -> int:
         batch_id = self._parse_post_response(response.json())
         if batch_id is None:
             raise AirflowException("Unable to parse the batch session id")
-        self.log.info("Batch submitted with session id: %d", batch_id)
+        self.log.info(f"Batch submitted with session id: {batch_id}")
```

Comment:
See https://docs.python.org/3/howto/logging.html#optimization

f-string is not the prefered choice for logging module

Reply:
Thanks for the comment @eladkal! So it will be better if we cast to int then?

```suggestion
        self.log.info("Batch submitted with session id: %d", int(batch_id))
```

Reply:
Why not just `%s` ?

Reply:
Indeed! Sounds better! 
Changed to `%s`! Thanks